webapp/channels/src/components/start_trial_form_modal/index.tsx

```diff
@@ -44,13 +43,32 @@ enum TrialLoadStatus {
     Failed = 'FAILED'
 }
 
-// Marker functions so i18n-extract doesn't remove strings
-t('ONE_TO_50');
-t('FIFTY_TO_100');
-t('ONE_HUNDRED_TO_500');
-t('FIVE_HUNDRED_TO_1000');
-t('ONE_THOUSAND_TO_2500');
-t('TWO_THOUSAND_FIVE_HUNDRED_AND_UP');
+defineMessages({
```

Comment:
I tried adding this inline to the enum below. I'm pretty sure that shouldn't be written using an enum because we also use it as the source of translation strings, but trying to change that ended up being far too much work to be worth cleaning it up a little